projects/ngx-script-loader/package.json

```diff
@@ -14,8 +14,11 @@
     "url": "https://github.com/muratcorlu/ngx-script-loader/issues"
   },
   "homepage": "https://github.com/muratcorlu/ngx-script-loader#readme",
+  "dependencies": {
+    "tslib": "^2.0.0"
+  },
   "peerDependencies": {
-    "@angular/common": ">=6.0.0 <9.0.0",
-    "@angular/core": ">=6.0.0 <9.0.0"
+    "@angular/common": ">=6.0.0 <12.0.0",
```

Comment:
Now I'm considering to write here just `">=11.0.0"` with a major version release of ngx-script-loader. So people who want to use with an older version of angular can continue with current version and in every major version, we can just support latest version of Angular. What do you think?